### PR TITLE
Feature: add delete functionality to the members page

### PR DESCRIPTION
**Pull request by ShoeheyOt**
## Overview

adding "DELETE member from a certain group" function
- add dropdown menu to the kebabu button
- display member delete dialog
- implement delete function (event handler/action and API client) including schema.
- issue #257 
## Screen Captures

[delete-member-fn.webm](https://github.com/nishiki-tech/nishiki-frontend/assets/142355969/50fb61d2-1a9e-43c9-a93d-fe89a1727fcf)

## Assignee Checklist:

- [X] The naming convention of the PR title is correct (See the comment at the top of this template)
- [X] The base branch is correct (See: [Types of Branches](https://github.com/nishiki-tech/nishiki-frontend/blob/develop/docs/CONTRIBUTING.md#types-of-branches))
- [X] The branch name follows the [Branch Naming Conventions](https://github.com/nishiki-tech/nishiki-frontend/blob/develop/docs/CONTRIBUTING.md#branch-naming-conventions)
- [X] The correct assignees and reviewers have been designated for this PR
- [X] The coding style follows the [Coding Style Guide](https://github.com/nishiki-tech/nishiki-frontend/blob/develop/docs/STYLEGUIDE.md)
- [X] All the related issues are associated with this PR
- [X] All criteria in the associated issues are met (please tick the checkboxes)
- [X] My changes do not generate new warnings or errors (especially in the console)

## Reviewer Checklist:

- [x] Double-check the "Assignee Checklist"
- [x] The code follows the generic best practices and our [Coding Style Guide](https://github.com/nishiki-tech/nishiki-frontend/blob/develop/docs/STYLEGUIDE.md)
- [x] The code is well-commented and easy to understand
- [x] The UI changes accurately reflect the provided design

### CODE REVIEW

**Review by nick-y-ito — APPROVED** [review]
@ShoeheyOt 
Thank you for your work. Although I have left one comment, there is nothing crucial to point out from me. Excellent job!!

File: src/features/groups/components/MemberCardList/MemberCardDropdownMenuTriggerButton/MemberCardDeleteButton/MemberCardDeleteDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      </DialogBody>
+      <DialogFooter>
+        <DialogClose asChild>
+          <Button
+            type="button"
+            variant="cancel"
+            size="sm"
+            onClick={() => {
+              onParentClose();
+            }}
```
[the remark this answers is not in the document]
Comment (ShoeheyOt) [reply src/features/groups/components/MemberCardList/MemberCardDropdownMenuTriggerButton/MemberCardDeleteButton/MemberCardDeleteDialog.tsx:72]: @nick-y-ito Yeah, you are totally right! I don't have to make it as arrow function.I will fix it!

File: src/components/page/MembersPage.tsx
Comment (kanta1207) [inline src/components/page/MembersPage.tsx:1]: Now this file has less code and easy to see! Thank you for organizing!

**Review by kanta1207 — APPROVED** [review]
LGTM! Sorry for the delay, and thank you for your work!

**Pull request merged**